### ISSUE TITLE
Implement Bulk Update

### ISSUE THREAD

**Issue by JoshDunnDev** [digitalbazaar/bedrock-resource-restriction#11]
Removed TODO for implementing a bulk update and adding an issue here. Bulk update still need to be implemented.

**Comment by aljones15** [comment]
API docs:

http://mongodb.github.io/node-mongodb-native/3.5/api/Collection.html#bulkWrite

http://mongodb.github.io/node-mongodb-native/3.5/api/Collection.html#updateMany

I believe bulkWrite might be needed in this case.

**Comment by dmitrizagidulin** [comment]
Implemented in PR #10, closing.

**Comment by aljones15** [comment]
@dmitrizagidulin I don't believe bulkUpdate was implemented in that PR only bulkInsert and bulkDelete.

**Comment by dmitrizagidulin** [comment]
@aljones15 sorry about that! :) Misread the title. Re-opening. :)

**Comment by aljones15** [comment]
no problem. if this isn't done by next week I can take it.
